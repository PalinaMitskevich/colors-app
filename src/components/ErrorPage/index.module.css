.errorContainer {
    text-align: center;
}

.errorCode {
    margin: 90px 0 30px;
    font-size: 110px;
}

.errorMessage {
    font-size: 40px;
    margin-bottom: 40px;
}

.errorDescription {
    font-size: 26px;
}
