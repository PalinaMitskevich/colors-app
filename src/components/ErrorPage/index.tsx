import React from 'react';
import {observer} from "mobx-react-lite";
import {appState} from "../../store";
import styles from './index.module.css'

const ErrorPage = observer(() => {

    return (
        <div className={styles.errorContainer}>
            <h1 className={styles.errorCode}>{appState.errorCode}</h1>
            <h2 className={styles.errorMessage}>WHOOPS!</h2>
            <h3 className={styles.errorDescription}>Something went wrong</h3>
        </div>
    );
});

export default ErrorPage;