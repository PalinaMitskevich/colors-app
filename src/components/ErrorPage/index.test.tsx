import ErrorPage from './index';
import { create } from 'react-test-renderer'

describe('ErrorPage', () => {
    it('Should render correctly', () => {
        const comp = create(<ErrorPage />)
        expect(comp).toMatchSnapshot()
    })
})