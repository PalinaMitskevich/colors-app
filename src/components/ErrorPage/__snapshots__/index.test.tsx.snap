// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage Should render correctly 1`] = `
<div
  className="errorContainer"
>
  <h1
    className="errorCode"
  />
  <h2
    className="errorMessage"
  >
    WHOOPS!
  </h2>
  <h3
    className="errorDescription"
  >
    Something went wrong
  </h3>
</div>
`;
